<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controllerForm</class>
 <widget class="QWidget" name="controllerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="btnHome">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>auto home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPos1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>pos 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPos2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>pos 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPos3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>pos 3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="grindTime_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5.00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="des_y">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="des_z">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="des_x">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="grindTime_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5.00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="grindTime_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5.00</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>30</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="x_dis">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="y_dis">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="z_dis">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="zInput">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>z input</string>
   </property>
  </widget>
  <widget class="QLabel" name="polishTime1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>polishing time 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="yInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>y input</string>
   </property>
  </widget>
  <widget class="QLabel" name="xInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>x input</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>polishing time 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>polishing time 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>current y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>current x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>current z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>current force</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>230</y>
     <width>149</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="boardComButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="boardBaudButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="arduinoComButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="arduinoBaudButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="polishTime1_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Board COM port</string>
   </property>
  </widget>
  <widget class="QLabel" name="polishTime1_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Board Baud rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="polishTime1_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arduino COM port</string>
   </property>
  </widget>
  <widget class="QLabel" name="polishTime1_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arduino Baud rate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QPushButton" name="actCamera">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate Camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="photoButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>350</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Take photo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deactCamera">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>310</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>DeAvtivate Camera</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
